docker/docker.go

```diff
@@ -225,15 +220,17 @@ func UseLocalOrPull(ctx context.Context, dClient *client.Client, hostname string
 }
 
 func RemoveContainer(ctx context.Context, dClient *client.Client, hostname string, containerName string) error {
-	err := dClient.ContainerRemove(ctx, containerName, types.ContainerRemoveOptions{})
+	err := dClient.ContainerRemove(ctx, containerName, types.ContainerRemoveOptions{Force: true})
 	if err != nil {
 		return fmt.Errorf("Can't remove Docker container [%s] for host [%s]: %v", containerName, hostname, err)
 	}
 	return nil
 }
 
 func StopContainer(ctx context.Context, dClient *client.Client, hostname string, containerName string) error {
-	err := dClient.ContainerStop(ctx, containerName, nil)
+	// define the stop timeout
+	stopTimeoutDuration := StopTimeout * time.Second
+	err := dClient.ContainerStop(ctx, containerName, &stopTimeoutDuration)
```

Comment:
what will docker do if the timeout is hit?

Reply:
docker will force stop the container, the default is 10 seconds